src/Library/demos/Drag and Drop/main.js

```diff
@@ -0,0 +1,64 @@
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+
+const list = workbench.builder.get_object("list");
+
+// Iterate over ListBox children
+for (const row of list) {
+  row.selectable = false;
+  let _drag_x;
+  let _drag_y;
+
+  const drag_source = new Gtk.DragSource();
```

Comment:
```suggestion
  const drag_source = new Gtk.DragSource({
    actions: Gtk.DragAction.MOVE,
  });
```

You need to set the drag action here. It's likely that the `Gtk.DropTarget` is ignoring actions from the source that don't match the target.

Reply:
ah okay, thank you!